search/median_search2.cpp

```diff
@@ -0,0 +1,133 @@
+/**
+ * @file
+ *
+ * @details
```

Comment:
There's still missing a brief description using `@brief`.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Ah, I didn't realize I need use @brief, sorry I'm still new. I believe my latest commit address this issue.